docs/data-cookbooks/setup.md

```diff
@@ -52,7 +52,7 @@ The schema for collections can be found [here](https://github.com/nasa/cumulus/t
 
 Providers generate and distribute input data that Cumulus obtains and sends to workflows. Schema for providers can be found [here](https://github.com/nasa/cumulus/tree/master/packages/api/models/schemas.js) in the object assigned to `module.exports.provider`. A few example provider configurations can be found [here](https://github.com/nasa/cumulus/tree/master/example/data/providers). Providers can be viewed, edited, added, and removed from the Cumulus dashboard under the "Providers" navigation tab. Additionally, they can be managed via the [providers api](https://nasa.github.io/cumulus-api/?language=Python#list-providers).
 
-**Break down of [s3_provider.json](https://github.com/nasa/cumulus/tree/tree/example/data/providers/s3_provider.json):**
+**Break down of [s3_provider.json](https://github.com/nasa/cumulus/tree/master/example/data/providers/s3_provider.json):**
```

Comment:
Yeah it's fine because the bucket actually gets replaced when the test data gets setup. It gets replaced with the user's internal bucket from the `config.yml`